demo/tree-view-demo.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { TreeView } from '../src';
+import { TreeItemData } from '../src/components/tree-view/tree-view';
+
+interface TreeViewDemoState {
+    selectedItem: Object;
+}
+
+export const treeData: TreeItemData[] = [
```

Comment:
The demo is base of documentation. It's not just some sample skizza on the side. Let the selected item influence something in the demo gui. Think about the children.